examples/getting_started/tf/nvflare_tf_getting_started.ipynb

```diff
@@ -0,0 +1,453 @@
+    "\n",
+    "            model = self.update_model(model, aggregate_results)\n",
+    "\n",
+    "            self.save_model(model)\n",
+    "\n",
+    "        self.info(\"Finished FedAvg.\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d24b6476-089a-4e9d-825b-07107bd5d84a",
+   "metadata": {},
+   "source": [
+    "### Client Code \n",
+    "Given a CIFAR10 [TensorFlow](https://www.tensorflow.org/) code example with a network defined at [src/tf_net.py](src/tf_net.py), we wish to adapt this centralized training code to something that can run in a federated setting.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c551053-5460-4d83-8578-796074170342",
+   "metadata": {},
+   "source": [
+    "On the client side, the training workflow is as follows:\n",
+    "1. Receive the model from the FL server.\n",
+    "2. Perform local training on the received global model\n",
+    "and/or evaluate the received global model for model\n",
+    "selection.\n",
+    "3. Send the new model back to the FL server."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c02bfc2a-783c-494f-9427-c38f40a2e870",
+   "metadata": {},
+   "source": [
+    "Using NVFlare's client API, we can easily adapt machine learning code that was written for centralized training and apply it in a federated scenario.\n",
+    "For a general use case, there are three essential methods to achieve this using the Client API :\n",
+    "- `init()`: Initializes NVFlare Client API environment.\n",
+    "- `receive()`: Receives model from the FL server.\n",
+    "- `send()`: Sends the model to the FL server."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9115ee07-d848-4a7c-99ad-64e20ab7093c",
+   "metadata": {},
+   "source": [
+    "With these simple methods, the developers can use the Client API\n",
+    "to change their centralized training code to an FL scenario with\n",
+    "five lines of code changes as shown below.\n",
+    "```python\n",
+    "    import nvflare.client as flare\n",
+    "    \n",
+    "    flare.init() # 1. Initializes NVFlare Client API environment.\n",
+    "    input_model = flare.receive() # 2. Receives model from the FL server.\n",
+    "    params = input_model.params # 3. Obtain the required information from the received model.\n",
+    "    \n",
+    "    # original local training code\n",
+    "    new_params = local_train(params)\n",
+    "    \n",
+    "    output_model = flare.FLModel(params=new_params) # 4. Put the results in a new `FLModel`\n",
+    "    flare.send(output_model) # 5. Sends the model to the FL server.  \n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "67432f44-4144-4347-8d74-e7f57e065a14",
+   "metadata": {},
+   "source": [
+    "The full client training script is saved in a separate file, e.g. [./src/cifar10_tf_fl.py](./src/cifar10_tf_fl.py) doing CNN training on the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset.\n",
+    "```python\n",
```

Comment:
In the [pt notebook](https://github.com/SYangster/NVFlare/blob/getting_started_notebooks/examples/getting_started/pt/nvflare_pt_getting_started.ipynb), we don't show the full client training script. Maybe just link to it here as well?